float32 sys_time
float64[] ef
float32[] q
float32[] dq
float32[] ddq
float32[] sac
float32[] u
bool accept
float32 bf 

